### ISSUE TITLE
infobox, or navbar collapsable

### ISSUE THREAD

**Issue by diablo6006** [StarCitizenTools/mediawiki-skins-Citizen#169]
## The Problem
can you keep old stile for navbox, with vector skin or Minerva, the title of collapsable nav bar width is 100% when collapsed, 
the attached files are screenshots that describes what I mean.
![infobox](https://user-images.githubusercontent.com/17708182/89931685-9890a500-dc1d-11ea-8539-abb40002c882.png)
![infobox2](https://user-images.githubusercontent.com/17708182/89931694-9e868600-dc1d-11ea-952b-f89074beea31.png)
![infobox3](https://user-images.githubusercontent.com/17708182/89931695-9f1f1c80-dc1d-11ea-8206-f96b7450d845.png)
![infobox4](https://user-images.githubusercontent.com/17708182/89931699-9fb7b300-dc1d-11ea-944d-b1adc6993ab5.png)

link to navbox:
https://en.wikipedia.org/wiki/Template:Navbox

**Comment by alistair3149** [comment]
The navbox styles are not included in skins like Citizen and Vector. It is defined elsewhere either on the wiki's common/skinname.css or inside the template.

**Comment by diablo6006** [comment]
Can you share those styles pls? as i tried to use styles listed under mediawiki:commons.css at wikipedia, but that gives the same result as in the screenshots above.

**Comment by alistair3149** [comment]
Yes the purple ones are exactly the old styles for Vector that you mentioned
